### ISSUE TITLE
Old motd doesn't get deleted

### ISSUE THREAD

**Issue by dewi-ny-je** [bderenzo/tinymotd#6]
I run the install steps for automatic installation and now when I log in I get multiple motd because when a new one is generated, the old one is not deleted. 

I run (as root):
```
./install -u myuser
./generate
```
I can show you my current services:
```
[Service]
Type=oneshot
ExecStart=/home/myuser/src/tinymotd/tinymotd/generate
StandardOutput=file:/home/myuser/src/tinymotd/tinymotd/data/motd
StandardError=file:/home/myuser/src/tinymotd/tinymotd/data/motd
```
Has something went wrong? I think I followed the instrutions provided.

**Comment by robinmensch** [comment]
Have you checked .bashrc ?

**Comment by dewi-ny-je** [comment]
it prints the current available motd... I'll reinstall once I have time

**Comment by bderenzo** [comment]
@dewi-ny-je You are right, there is a problem with the usage of systemd `StandardOutput=file:/...` directive. It does not truncate the file [see systemd issue #8983](https://github.com/systemd/systemd/issues/8983). So, there are cases where it is possible to have parts of 2 different executions in the motd file. A solution has been implemented in [this pull request #18250 of systemd](https://github.com/systemd/systemd/pull/18250). I just updated the titnymotd systemd service accordingly.
